### PR TITLE
fix visible scrollbar in firefox

### PR DESCRIPTION
**Pull request by ueen**
see here
https://www.w3schools.com/howto/howto_css_hide_scrollbars.asp

fixes https://github.com/schlagmichdoch/PairDrop/issues/240

### CODE REVIEW

File: public/styles/styles-main.css [hunk 1]
```diff
@@ -120,6 +120,7 @@ html {
     display: flex;
     align-items: center;
     justify-content: center;
+    scrollbar-width: none;
```
Comment (schlagmichdoch) [inline public/styles/styles-main.css:123]: In the issue you state the following:

> its part of the div with `id=center`

But the css selector you used is for elements with `class=center`.
Are you sure this is correct and does indeed fix your issue? As I cannot reproduce the issue I am dependent on your testing!

Comment (schlagmichdoch) [reply public/styles/styles-main.css:123]: Also, are you sure this is sufficient? Is the issue specific to Firefox on MacOS or have you successfully reproduced it on any other Browser or OS? If so, we would need to add the following (according to your w3school link):
```css
/* Hide scrollbar for Chrome, Safari and Opera */
.example::-webkit-scrollbar {
  display: none;
}
```
Where `.example` must be replaced by the correct selector.

**Review by schlagmichdoch — CHANGES_REQUESTED** [review]

File: public/styles/styles-main.css (hunk 1 above)
Comment (ueen) [reply public/styles/styles-main.css:123]: Are there other elements that use the center class? Did not see it in Safari :)

Comment (schlagmichdoch) [reply public/styles/styles-main.css:123]: The class center is used to center its element via flexbox. The div with id center is not centered so it does not have the class center.

Please test whether your code works by adding it to the `#center` selector

**Comment by ueen** [comment]
https://github.com/schlagmichdoch/PairDrop/pull/251

**Pull request closed without merging**